<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicomViewer</class>
 <widget class="QWidget" name="DicomViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dicom Viewer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="tool_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>dcmviewer_tool_bg</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="fileBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;检索并加载文件夹图像&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>文件操作</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/open.png</normaloff>:/png/open.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer_begin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="exportBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;导出图像&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>导出</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/share.png</normaloff>:/png/share.png</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="gridBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;视图分格&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>视窗分格</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/grid.png</normaloff>:/png/grid.png</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="annoBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;显示/隐藏角注&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>标签</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/anno.png</normaloff>:/png/anno.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="adjustBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;调整窗宽窗位&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>窗宽窗位</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/adjust.png</normaloff>:/png/adjust.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="panBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;图像平移&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>图像移动</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/move.png</normaloff>:/png/move.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="zoomBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;图像缩放&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>图像缩放</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/zoom.png</normaloff>:/png/zoom.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="marksBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;标注和测量&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>标注测量</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/measure.png</normaloff>:/png/measure.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="flipBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;图像翻转&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>图像翻转</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/flip.png</normaloff>:/png/flip.png</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pluginBtn">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;三维重建及MPR&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>三维重建</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/reconstruction.png</normaloff>:/png/reconstruction.png</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="fullScreenBtn">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="text">
           <string>全屏设置</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/dicomviewer.qrc">
            <normaloff>:/png/full_screen.png</normaloff>:/png/full_screen.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="ViewContainerWidget" name="viewContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="ThumbnailBarWidget" name="thumbnailBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>157</width>
        <height>604</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewContainerWidget</class>
   <extends>QWidget</extends>
   <header location="global">view/KissView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ThumbnailBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">view/KissView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/dicomviewer.qrc"/>
  <include location="../resources/dicomviewer.qrc"/>
  <include location="../resources/dicomviewer.qrc"/>
  <include location="../resources/dicomviewer.qrc"/>
 </resources>
 <connections/>
</ui>
